src/upgrade.js

```diff
@@ -62,8 +62,19 @@ Upgrade.getAll = async function () {
 Upgrade.appendPluginScripts = async function (files) {
 	// Find all active plugins
 	const activePlugins = await plugins.getActive();
-	activePlugins.forEach((plugin) => {
-		const configPath = path.join(paths.nodeModules, plugin, 'plugin.json');
+	for (const plugin of plugins) {
```

Comment:
This seems like a typo, should be `for (const plugin of activePlugins)` I think.